Spigot-Server-Patches/0406-Fixed-MC-156852.patch

```diff
@@ -11,18 +11,51 @@ issue in 1.8-1.12.
 
 Originally solved by Gnembon on MC-5694 at bugs.mojang.com
 
+From: Zean Rivera <zean@zean.org>
+Date: Sat, 7 Sep 2019 03:11:05 -0700
+Subject: [PATCH] Add a flag, "disable-ghost-block-fix", to allow for turning off the GhostBlockFix https://github.com/PaperMC/Paper/pull/2396
+
+The reason for this is that this fix re-introduces
+this bug https://bugs.mojang.com/browse/MC-156013
+
+This gives server owners a way to opt out of the GhostBlockFix.
+It defaults to false, which is keeping the GhostBlockFix.
+
```

Comment:
We do not add duplicate patch headers to files. Remove this. Git does a fine enough job keeping track of who did what without adding multiple headers to files.